site/_docs/reference.md

```diff
@@ -2331,6 +2332,7 @@ Note:
 * `JSON_LENGTH` defines a JSON value's length as follows:
   * A scalar value has length 1;
   * The length of array or object is the number of elements is contains.
+* `EXTRACTVALUE` returns null if any argument is null
```

Comment:
you don't need to say this. this is default behavior for SQL functions.

Reply:
Done :)